eventsource-client/src/client.rs

```diff
@@ -29,14 +29,17 @@ use tokio::{
     time::Sleep,
 };
 
-use super::config::ReconnectOptions;
-use super::decode::Decoded;
-use super::error::{Error, Result};
+use crate::config::ReconnectOptions;
```

Comment:
We need a linter or formatter that keeps all of these together and prefers crate to super (or whichever we want).